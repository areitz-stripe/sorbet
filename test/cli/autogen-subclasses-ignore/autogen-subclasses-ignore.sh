#!/bin/bash
set -eu

echo "--- autogen-subclasses-ignore ---"
main/sorbet --silence-dev-message --stop-after=resolver -p autogen-subclasses \
  --autogen-subclasses-parent=Opus::Mixin \
  --autogen-subclasses-parent=Opus::Parent \
  --autogen-subclasses-parent=Opus::SafeMachine \
  --autogen-subclasses-parent=Chalk::ODM::Model \
  --autogen-subclasses-ignore=test \
  test/cli/autogen-subclasses/a.rb
