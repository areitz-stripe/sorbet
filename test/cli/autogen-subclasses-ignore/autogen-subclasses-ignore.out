--- autogen-subclasses-ignore ---
Opus::Mixin
Opus::Parent
Opus::SafeMachine
Chalk::ODM::Model
 Chalk::ODM::Private::Lock
